Content: New alien species: the Incipias (#7230)
endless-sky/endless-sky: data/incipias/first contact.txt

		event "Hai preparation time" 4

event "Hai preparation time"

mission "wanderer peace prove"
	landing
	name "Prove of Peace"
	source
		government "Wanderer"
		not planet "Vara K'chrai"
	destination "Vara K'chrai"
	to offer
		and
			has "searching for a prove"
			and
				not "coalition peace prove: offered"
				"reputation: Wanderer" >= 5
	on offer
		conversation
			`Now that you are on a wanderer world you decide this would be a good time for asking them about the prove the Hai requested from you. You head for a near embassy.`
			`    At the embassy an older Wanderer ask you what you are there for. You remember the Hai telling you that you should not tell the other species about the Incipias, so you search for something else to tell them.`
			choice
				"I am searching for something unique that presents your species best."
					goto unique
				"I am gathering proves of friendship from all over the galaxy."

> 			`   "That sounds interesting, let me think about it." The Wanderer becomes silent for some time. Then he says, "I got an idea, but I need to ask some people about it first. Please wait here for a second." He leaves the room without waiting for your respond. Forced to wait for him, you ask yourself what this gift of friendship of the wanderers could be.`

Reviewer: ```suggestion
			`   "That sounds interesting, let me think about it." The Wanderer becomes silent for some time. Then he says, "I have an idea, but I need to ask some people about it first. Please wait here for a second." He leaves the room without waiting for your respond. Forced to wait for him, you ask yourself what this gift of friendship of the wanderers could be.`
```
The dialogue with the Wanderer should probably be adjusted to include the translation device's suggestions (for examples, look at the Wanderer missions).

				goto c1
			label unique
			`    "Why are you searching for something like this? That is an quite unusual request."`
			choice
				"I am a passionate collector."
				"I want something to remember you."

> 			`    "Let me think about this. What would present our species best..." The Wanderer gets silent while thinking. Than he suddenly stands up, "If I get the permission to give you this, you will get a perfect item that presents our species best. Let me go check it." The Wanderer leaves the room.`

Reviewer: ```suggestion
			`    "Let me think about this. What would represent our species best..." The Wanderer gets silent while thinking. Than he suddenly stands up, "If I get the permission to give you this, you will get a perfect item that represents our species best. Let me go check it." The Wanderer leaves the room.`
```
Again, the Wanderer's dialogue should use the translation machine suggestions

			label c1

> 			`    Time is passing and you get a little bit nervous, but after some time the wanderer finally returns. He immedeatly begins to speak. "I just got the permission", he says "You will have to travel to <destination> to get this. You will receive it at the embassy there, I already contacted some people there." You ask him what it his but all he tells you is "It is a surprise, you will see when you get there. Take this." He hands you a small card. "This will tell the people there that you are the right one." He says goodbye to you and you leave the embassy.`

Reviewer: ```suggestion
			`    Time passes and you get a little bit nervous, but after some time the Wanderer finally returns. He immediately begins to speak. "I just got permission", he says "You will have to travel to <destination> to get this. You will receive it at the embassy there, I already contacted some people there." You ask him what it his but all he tells you is "It is a surprise, you will see when you get there. Take this." He hands you a small card. "This will tell the people there that you are the right one." He says goodbye to you and you leave the embassy.`
```

				accept
	on complete
		conversation
			`    After you arrived at Vara K'chrai, you directly head to the embassy excited what the Wanderers will give you. At the embassy you show the small card the Wanderer gave you and a woman guides you to a separate room, where three Wanderer are standing around a table. On the table is a small plant in a pot. But you have never seen a plant like this before on any Wanderer world. One of the three begins to talk. "This plant on the table there is our gift to you. It is a genmanipulated tree, that will survive under mostly any circumstances, it is something our species is specialized on and it will show our friendship in a very unique way. We hope you like it." Surprised by this unusual gift you thank the wanderers. "It needs nearly no water, so you do not have to care a lot about it, it survives on spaceship so you can take it with you anywhere." You again say thanks to them and then you leave.`

> 			`    You now have a prove of the Wanderers friendship. You can show it to the Hai now.`

Reviewer 2: ```suggestion
			`    You now have a proof of the Wanderers friendship. You can show it to the Hai now.`
```

> mission "coalition peace prove"

Reviewer: ```suggestion
mission "coalition peace proof"
```

	landing

> 	name "Prove of Peace"

Reviewer: ```suggestion
	name "Proof of Peace"
```

	source
		government "Heliarch"
	to offer
		has "Coalition: First Contact: done"

> 		not "wanderer peace prove: offered"

Reviewer: ```suggestion
		not "wanderer peace proof: offered"
```

	on offer
		conversation

> 			`As the Heliarch have a relationship to you feel not really like friendship but something in that direction, you decide to ask them for a prove of this relationship, to give to the Hai.`

Reviewer: ```suggestion
			`As the Heliarch have a relationship to you feel not really like friendship but something in that direction, you decide to ask them for a proof of this relationship, to give to the Hai.`
```

> 			`    You approach a nearby Heliarch and tell him you want to ask the government about a personal matter. He gives you directions to an office that handles maters like yours. You thank the Heliarch and make your way to the office.`

Reviewer: ```suggestion
			`    You approach a nearby Heliarch and tell him you want to ask the government about a personal matter. He gives you directions to an office that handles matters like yours. You thank the Heliarch and make your way to the office.`
```

			`    When you arrive there, you end up in a pretty long waiting line. It seems that the Heliarch are never short of work. You patiently line yourself up in the line and get ready to spend some time in the line.`

> 			`    After some time has passed it is finally your turn to tell your matter. You explain that you would like to have something that displays the good relationship between the Coalition and you. The Heliarch is quite surprised and says: "An interesting request of yours that is. Why you want this, may I ask?"`

Reviewer: ```suggestion
			`    After some time has passed it is finally your turn to discuss your matter. You explain that you would like to have something that displays the good relationship between the Coalition and you. The Heliarch is quite surprised and says: "An interesting request of yours that is. Why you want this, may I ask?"`
```

			choice

> 				"I am gathering proves of friendship from all over the galaxy."

Reviewer: ```suggestion
				"I am gathering proofs of friendship from all over the galaxy."
```

					goto cages
				"I want to show that species do not have to be in cages."
			`    The Heliarch seems very happy when you say that, he says "A good reason that is, here you wait, while some people above me about this I ask." He turns around and goes away.`
				goto c1
			label cages
			`    "An unusual project that is, what the reason for this is?"`
			choice
				"I am a passionate collector."
				"I want something to remember you, to remember peace is possible."

> 			`    "Yor  project understand I can. The people over me about your request let me ask." He turns around and goes away.`

Reviewer: ```suggestion
			`    "Your project understand I can. The people over me about your request let me ask." He turns around and goes away.`
```

			label c1

> 			`    When noises in the line behind you are growing louder he finally comes back. With him he carries a small box. "A decent gift for you I got." He opens the box and in there a small gem like no one you have seen before is laying, reflecting the light in green, red and blue shades. "This gem a fusion of three others it is, an emerald for the Saryds, a ruby for the Kimek and a sapphire for the Arachi. A fusion of our three species it is, you could say. This gem representing our species best it is. You appreciate this gift I hope." You are really thankful for this gift and thank the Heliarch a lot. But then you have to leave because there is still a line behind you.`

Reviewer: ```suggestion
			`    When noises in the line behind you are growing louder he finally comes back. With him he carries a small box. "A decent gift for you I got." He opens the box and in it lies a small gem like none you have seen before, reflecting the light in green, red and blue shades. "This gem a fusion of three others is, an emerald for the Saryds, a ruby for the Kimek and a sapphire for the Arachi. A fusion of our three species it is, you could say. This gem representing our species best it is. You appreciate this gift I hope." You are really thankful for this gift and thank the Heliarch a lot. But then you have to leave because there is still a line behind you.`
```

				accept

mission "Incipias: First Contact IV"
	landing
	name "A Hai delegation"
	to offer
		has "event: Hai preparation time"
	source Mirrorlake
	destination "Spec Inci"
	on offer
		conversation

> 			`At the spaceport you see a small group of Hai diplomats, one of them is familiar to you from the embassy. You walk towards them and greet them. One of them says, "I hope for a save journey on your ship Captain." You answer, "The system we are travlling to is not far from here." He says, "Than not more is to be discussed, bring us to the Quarg ringworld, so that we can talk to them." You agree and get your ship ready.`

Reviewer: ```suggestion
			`At the spaceport you see a small group of Hai diplomats, one of them is familiar to you from the embassy. You walk towards them and greet them. One of them says, "I hope for a safe journey on your ship Captain." You answer, "The system we are travelling to is not far from here." He says, "Then no more is to be discussed, bring us to the Quarg ringworld, so that we can talk to them." You agree and get your ship ready.`
```

				accept
	on complete
		conversation
			`When you arrive at the ringworld the Hai take the lead. The few Quarg around seem surprised by the presence of the Hai. The Hai talk to one of the Quarg and it shows you the way.`

> 			`    Once again you are overwhelmed by the size of a Quarg ringworld. It takes you some time tto get to your destination. When you get to the location the Quarg told you to go, one of the Hai turns towards you. "It will be the best if we talk to them without you being present. I hope you understand that." He tells you. Surprised by this you just nod in agreement. The Hai walk into the room and leave you alone.`

Reviewer: ```suggestion
			`    Once again you are overwhelmed by the size of a Quarg ringworld. It takes you some time to get to your destination. When you get to the location the Quarg told you to go, one of the Hai turns towards you. "It will be the best if we talk to them without you being present. I hope you understand that." He tells you. Surprised by this you just nod in agreement. The Hai walk into the room and leave you alone.`
```

mission "Incipias: First Contact V"
	landing
	invisible
	to offer
		has "Incipias: First Contact IV: done"
	on enter Dom'us
		conversation
			`This time as you enter this system, you are prepared. You stop your ship and pull out the translation device the Quarg gave you. Then you stand there and think, what would be the best first contact message?`
			choice
				"Hello Incipiae, I am <first> <last> of the human race a space ship captain."
				"Greetings strangers, I come to spread peace."
				"Hello beings of another kind, I want to communicate with you."
			`    At first nothing happens. Then your scanners register a huge raise in communication between the alien ships, their planet and their spacestation. At first it seems to be very chaotic but then slowly the spacestation dominates the communication. Then there is nothing.`
			choice
				"[wait for some time]"

> 			`    After some time the space station opens a communication channel to you. A creature like you never seen one before appears on the screen. Formed like a balloon, with some weird shaped extremeties and four eyes? at the bottom.It is floating in front of a blank wall. You remember what the Quarg told you and hold the tablet in the direction of the screen. Something on the surface of the creature begins flickering and words start appearing on the tablet screen. "Welcome, [stranger, traveller, alien] [I, we] are [happy, surprised, scared] by your visit." It seems like this is the difference the Quarg talked about, the translation device is trying to combine words to give you the actual meaning. More words appear, "[I, we] see another [intelligence, feeling one] the first time, please [explain, show, tell us] the reason for your presence."`

Reviewer: ```suggestion
			`    After some time the space station opens a communication channel to you. A creature like you never seen one before appears on the screen. Formed like a balloon, with some weird shaped extremeties and four eyes? at the bottom. It is floating in front of a blank wall. You remember what the Quarg told you and hold the tablet in the direction of the screen. Something on the surface of the creature begins flickering and words start appearing on the tablet screen. "Welcome, [stranger, traveller, alien] [I, we] are [happy, surprised, scared] by your visit." It seems like this is the difference the Quarg talked about, the translation device is trying to combine words to give you the actual meaning. More words appear, "[I, we] see another [intelligence, feeling one] the first time, please [explain, show, tell us] the reason for your presence."`
```

			choice
				"I am exploring the limits of space."
					goto limits
				"I am searching for new species and technologies"

> 			`    The Incipias listens to you and then it looks like they are looking to someone on its right you do not see. After a short moment it turns towards you again. It starts flickering "We do not think we have [better, more advanced] technologies than [you, your kind]. But [I, we] look forward to a [benifical, peaceful] contact with [you, your kind]. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this adventure] further.`

Reviewer: ```suggestion
			`    The Incipias listens to you and then it looks like they are looking to someone on its right you do not see. After a short moment it turns towards you again. It starts flickering "We do not think we have [better, more advanced] technologies than [you, your kind]. But [I, we] look forward to a [benefical, peaceful] contact with [you, your kind]. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this adventure] further.`
```

> 			`    The Incipias listens to you and then it looks like they are looking to someone on its right you do not see. After a short moment it turns towards you again. It starts flickering "We do not think we have [better, more advanced] technologies than [you, your kind]. But [I, we] look forward to a [benefical, peaceful] contact with [you, your kind]. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this adventure] further.`

Reviewer 3: ```suggestion
			`    The Incipias listens to you and then it looks like they are looking to someone on its right you do not see. After a short moment it turns towards you again. It starts flickering "We do not think we have [better, more advanced] technologies than [you, your kind]. But [I, we] look forward to a [beneficial, peaceful] contact with [you, your kind]. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this adventure] further.`
```

				goto c1
			label limits
			`    The Incipias listens to you and then it looks like it is looking to someone on their right you do not see. After a short moment it turns towards you again. It starts flickering "[I, we] are happy to [greet, meet, see] you traveller. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this meeting] further.`
			label c1
			`    Some basic information about the spacestation is send to your display and the communication channel gets closed.`
			action

> 				"reputiation: Hicemus" = 1

Reviewer 2: ```suggestion
				"reputation: Hicemus" = 1
```

				"reputation: Incipias" = 1

mission "Incipias: First Contact VI"
	landing
	invisible
	to complete
		never

> 	source "Pon'tes"

Reviewer 2: no to offer?

Author: You can only land after you got a better rep

Reviewer 2: Okay but its number 6?
Also did you make sure one cant farm rep by refueling?

Author: I am not sure what you mean?

Author: > Okay but its number 6? Also did you make sure one cant farm rep by refueling?

not really, they will never run out of fuel, becuase they do not use fuel

			action
				set "hicemus knowledge: hyperlanes"
				"hicemus knowledge" = 0
			choice
				"[show them a map of human systems]"
					goto map
				"[show them the location of Sol in the Galaxy]"

> 			`    You show them a picture of the galaxy and point out where the Sol system is and where the Dom'us system is. The Incipias move closer to see and you notice some excitement in their rows. Their speaker turns towards you "A [long, unexpected distance is between our [systems, homes], [you, your kind] has to be [very, far more] advanced." You type in the tablet, "No, it is only a few of us who can travell that far between stars. My kind has not yet the technology to travell that far, I got this technology from another species." Again some rumor starts between the Incipias. "So we and [your kind] are not [the only ones, alone in space]?" Their speaker asks you.`

Reviewer: ```suggestion
			`    You show them a picture of the galaxy and point out where the Sol system is and where the Dom'us system is. The Incipias move closer to see and you notice some excitement in their rows. Their speaker turns towards you "A [long, unexpected distance is between our [systems, homes], [you, your kind] has to be [very, far more] advanced." You type in the tablet, "No, it is only a few of us who can travel that far between stars. My kind has not yet the technology to travel that far, I got this technology from another species." Again some rumor starts between the Incipias. "So we and [your kind] are not [the only ones, alone in space]?" Their speaker asks you.`
```
